localization/pose_initializer/media/pose_initialization.drawio.svg

```diff
@@ -0,0 +1,553 @@
+<svg
```

Comment:
The original file is here so please refer to it directly instead of copying it.
https://github.com/autowarefoundation/autoware.universe/blob/main/system/default_ad_api/document/images/localization.drawio.svg

Reply:
Thank you.
I could not come up with a better idea than referring to the file by relative path like this: https://github.com/autowarefoundation/autoware.universe/pull/2635/commits/74582032b45b3b21f165fb1317661eebe939e8fc.
Let me know if you have any better idea.

Reply:
As far as I know it's the only way. The svg file is no longer needed.

Reply:
Right, thank you. Fixed: https://github.com/autowarefoundation/autoware.universe/pull/2635/commits/f0001b1fa8e6bedc1dd343fa1ac226306ec1d64a